Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    redis (4.6.0)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
```

Comment:
### [rspec-core](https://github.com/rspec/rspec-core)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 2.11.1 → 3.11.0

[_[change-log](https://github.com/rspec/rspec-core/blob/v3.11.0/Changelog.md), [source-code](https://github.com/rspec/rspec-core), [gem-diff](https://my.diffend.io/gems/rspec-core/2.11.1/3.11.0)_]





<details>
<summary>Commits</summary>

A change of **3788** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-core/compare/v2.11.1...v3.11.0).

These are the first 10 commits:
- (e894c64) [Fix treatment of rake task option as an Array.](https://github.com/rspec/rspec-core/commit/e894c641d10435765f2b0ed3a07c5ac652db534f)
- (50e9c7f) [Merge pull request #2704 from marcandre/rspec_opt_array](https://github.com/rspec/rspec-core/commit/50e9c7fa49228ae0e64d872e422c54e6840f34c5)
- (002ebcb) [Changelog for #2704](https://github.com/rspec/rspec-core/commit/002ebcb65324bca7b3ee044e22f4c02133fce14d)
- (a5a6a26) [Add information about hook defaults to cucumber scenario](https://github.com/rspec/rspec-core/commit/a5a6a2612c3b615081a3befa6b0d061d2bf3e94a)
- (dd3b2ab) [Merge pull request #2708 from rspec/add-default-hook-docs](https://github.com/rspec/rspec-core/commit/dd3b2abe63b0dbd4c5c4131275a858d63cddab0e)
- (5e0b12d) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-core/commit/5e0b12ddd5bde88811ff1ef3161b5e68f994d439)
- (464b848) [Merge pull request #2709 from rspec/update-travis-build-scri…](https://github.com/rspec/rspec-core/commit/464b8488e80a90c5c1a76b476c1b814f79fc8efb)
- (0672a8e) [Updated travis build scripts (from rspec-dev)](https://github.com/rspec/rspec-core/commit/0672a8e0d5020e168bac30a7608a6711d5ac2403)
- (73c27f4) [Merge pull request #2714 from rspec/update-travis-build-scri…](https://github.com/rspec/rspec-core/commit/73c27f4131a1b2d91704e7ccc07b9bace861b4a4)
- (6a5d667) [Add scenario explaining example failures do not affect hooks](https://github.com/rspec/rspec-core/commit/6a5d667019e7d1e7d7d3452d29311a3e6e6afbac)


</details>